### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by IcedMonk**
## Update to copyright year
- Updated in build.ts file

### CODE REVIEW

File: script/build.ts [hunk 1]
```diff
@@ -199,7 +199,7 @@ function packageApp() {
       new RegExp('/\\.git($|/)'),
       new RegExp('/node_modules/\\.bin($|/)'),
     ],
-    appCopyright: 'Copyright © 2017 GitHub, Inc.',
+    appCopyright: 'Copyright © 20123 GitHub, Inc.',
```
Comment (andrewhong04) [inline script/build.ts]: The year is actually 2023.

File: script/build.ts [hunk 2]
```diff
@@ -199,7 +199,7 @@ function packageApp() {
       new RegExp('/\\.git($|/)'),
       new RegExp('/node_modules/\\.bin($|/)'),
     ],
```
Comment (yks-ny) [inline script/build.ts:201]: macosは不正使用の端末です。MACを買ったこともない

File: script/build.ts (hunk 1 above)
Comment (sergiou87) [reply script/build.ts]: As @novialriptide said, please fix the typo! 🙏

**Review by sergiou87 — CHANGES_REQUESTED** [review]

**Review by sergiou87 — APPROVED** [review]
Looks good now, thanks! ✨

**Pull request merged**